framework/src/main/java/org/tron/core/services/http/GetEnergyPricesServlet.java

```diff
@@ -1,36 +1,31 @@
 package org.tron.core.services.http;
 
-import com.alibaba.fastjson.JSONObject;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
+import org.tron.api.GrpcAPI.ResourcePricesResponseMessage;
 import org.tron.core.Wallet;
 
-
 @Component
 @Slf4j(topic = "API")
 public class GetEnergyPricesServlet extends RateLimiterServlet {
 
   @Autowired
   private Wallet wallet;
 
+  @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response) {
     try {
-      String reply = wallet.getEnergyPrices();
-      if (reply != null) {
-        JSONObject jsonObject = new JSONObject();
-        jsonObject.put("prices", reply);
-        response.getWriter().println(jsonObject);
-      } else {
-        response.getWriter().println("{}");
-      }
+      ResourcePricesResponseMessage reply = wallet.getEnergyPrices();
+      response.getWriter().println(reply == null ? "{}" : JsonFormat.printToString(reply));
```

Comment:
If you change the return type in `Wallet` and do not check the return result, the reply is never null?

Reply:
<img width="767" alt="image" src="https://github.com/tronprotocol/java-tron/assets/29877167/06314cd9-0954-462f-83b3-f351005cd333">
Returns null when an exception occurs.